lib/open_project/object_linking.rb

```diff
@@ -147,5 +155,15 @@ def url_to_attachment(attachment, only_path: true)
         v3_paths.url_for(:attachment_content, attachment.id)
       end
     end
+
+    def url_to_file_link(file_link, only_path: true)
+      v3_paths = API::V3::Utilities::PathHelper::ApiV3Path
```

Comment:
if the comment above about the module breaking when including the module is still true, extracting the v3_paths into a method that is then called would be the next best thing I think.

Reply:
Including the `PathHelper` seemed safe,but we would still need to prefix the calls with `ApiV3Path` which is shorter but I opted for the method as, if the breaking is subtle, it would be hard to me identifying it.